presentation-compiler/test/dotty/tools/pc/tests/highlight/DocumentHighlightSuite.scala

```diff
@@ -1462,5 +1462,106 @@ class DocumentHighlightSuite extends BaseDocumentHighlightSuite:
         |""".stripMargin
     )
 
+  @Test def i3053 =
+    check(
+      """import Aaaa.*
+        |
+        |def classDef2(cdef: List[Int]): Int = {
+        |  def aaa(ddef: Thicket2): List[Int] = ddef match {
+        |    case Thicket2(_) => ???
+        |  }
+        |
+        |  /** Does `tree' look like a reference to AnyVal? Temporary test before we have
```

Comment:
Could we skip the scaladoc actually, or make them a bit shorter?

Reply:
The problem is that the span is too big, so all this comments to make the method extra long are needed.

Reply:
Ach, makes sense. Thanks for explaining!

Reply:
It definitely falls under "things I would not have anticipated"!